### ISSUE TITLE
Resource alicloud_slb missing options for MasterZoneId and SlaveZoneId

### ISSUE THREAD

**Issue by nobjohns** [terraform-providers/terraform-provider-alicloud#798]
### Terraform Version
Terraform v0.11.11

### Affected Resource(s)
- alicloud_slb

### Terraform Configuration Files
```hcl
resource "alicloud_slb" "test" {
  name          = "test"
  vswitch_id    = "<removed>"
  internet      = false
  specification = "slb.s2.small"
```

### Expected Behavior
Resource alicloud_slb shoud have option to select Master and Slave Availability Zones
### Actual Behavior
Resource alicloud_slb don't have option to select Master and Slave Availability Zones
### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by xiaozhu36** [comment]
HI @nobjohns I am sorry for late reply. It has been fixed by #887

**Comment by nobjohns** [comment]
@xiaozhu36 Thanks, waiting for the new release with fix

**Comment by xiaozhu36** [comment]
Fixed by release 1.36.0
